api_types.ts

```diff
@@ -460,7 +460,7 @@ export interface Network {
 }
 
 /** The HTTP methods (verbs) supported by the fetcher. */
-export const ValidFetchMethods = ['GET', 'PATCH', 'POST', 'PUT', 'DELETE'] as const;
+export const ValidFetchMethods = ['GET', 'PATCH', 'POST', 'PUT', 'DELETE', 'HEAD'] as const;
```

Comment:
I think you also need some changes in the coda repo.

Reply:
yeah going to link that pr before merging this